### ISSUE TITLE
Gem issues reported in 1.7.2

### ISSUE THREAD

**Issue by ericfranz** [OSC/ondemand#316]
Two separate issues reported.

1. Both Kinan and Ariel try to access the Job Composer from the Dashboard and get this error:

```
Per user setup failed for script at /var/www/ood/apps/sys/myjobs/./bin/setup-production for user kalattar with output: Ignoring eventmachine-1.2.7 because its extensions are not built. Try: gem pristine eventmachine --version 1.2.7
Ignoring json-1.8.6 because its extensions are not built. Try: gem pristine json --version 1.8.6
Ignoring nokogiri-1.10.3 because its extensions are not built. Try: gem pristine nokogiri --version 1.10.3
Ignoring thin-1.7.2 because its extensions are not built. Try: gem pristine thin --version 1.7.2
bundler: failed to load command: ./bin/setup-production (./bin/setup-production)
Bundler::GemNotFound: Could not find rake-13.0.0 in any of the sources
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/spec_set.rb:91:in `block in materialize'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/spec_set.rb:85:in `map!'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/spec_set.rb:85:in `materialize'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/definition.rb:170:in `specs'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/definition.rb:237:in `specs_for'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/definition.rb:226:in `requested_specs'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/runtime.rb:108:in `block in definition_method'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/runtime.rb:20:in `setup'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler.rb:107:in `setup'
  /opt/ood/ondemand/root/usr/share/gems/2.5/gems/bundler-1.17.3/lib/bundler/setup.rb:20:in `<top (required)>'
  /opt/rh/rh-ruby25/root/usr/share/rubygems/rubygems/core_ext/kernel_require.rb:59:in `require'
  /opt/rh/rh-ruby25/root/usr/share/rubygems/rubygems/core_ext/kernel_require.rb:59:in `require'
```

2. When trying to use Bundle Install from the developer web interface it fails:

![image](https://user-images.githubusercontent.com/512333/69449998-5e8f1400-0d19-11ea-8ad9-853ebd6304d7.png)

**Comment by KinanAlAttar** [comment]
I also have ~/.asdf installed with a local Ruby install

**Comment by treydock** [comment]
This is caused by `~/.bundle/config`. I removed mine and it worked. My guess is the wrong bundler is getting used.

**Comment by KinanAlAttar** [comment]
That worked for me. I can now open Job Composer. Thanks.

**Comment by treydock** [comment]
Based on path it's correct bundler, but somehow `~/.bundle/config` is causing problems

**Comment by treydock** [comment]
So in Docker I noticed `http://localhost:9080/pun/sys/myjobs` works but the error URL is actually `http://localhost:9080/pun/sys/dashboard/apps/show/myjobs` which is what I get when I open `My Jobs` from the menu.

**Comment by treydock** [comment]
AH! During execution of `setup-production` we put `myjobs/bin` into $PATH. There is a `bundle` script in that directory that maybe screwing things up when we do `bundle exec`.

**Comment by treydock** [comment]
This was fixed in #318
